docs/4.3/enterprise/workflow/ssh_approval_jira_server.md

```diff
@@ -1,12 +1,12 @@
 # Teleport Jira Plugin Setup
 
-If you're using Jira Cloud or Jira Server to manage your projects, you can also use it to monitor, approve, deny, or discuss Teleport permission requests. This quickstart will walk you through the setup.
+This guide will talk through how to setup Teleport with Pagerduty.   Teleport ↔ Pagerduty integration  allows you to treat Teleport access and permission requests as Pagerduty incidents — notifying the appropriate team, and approve or deny the requests via Pagerduty special action.
```

Comment:
Spacing is weird here, lots of double/triple spaces. Also the `↔` character might not exist in some fonts.